pkg/minikube/cluster/cluster.go

```diff
@@ -170,8 +170,13 @@ type KubernetesConfig struct {
 }
 
 // StartCluster starts a k8s cluster on the specified Host.
-func StartCluster(h sshAble) error {
-	commands := []string{stopCommand, GetStartCommand()}
+func StartCluster(h sshAble, d drivers.Driver) error {
+	ipStr, err := d.GetIP()
```

Comment:
What do you think about pulling this up to the start.go file, adding ip to KubernetesConfig, and passing KubernetesConfig into StartCluster/GetStartCommand?

I ask because we're probably going to need to start passing KubernetesConfig through here anyway, to allow customization of Kubernetes flags in #529